Automattic/wp-calypso
# WIP: Add FSE Block Pattern validation tests

#### Changes proposed in this Pull Request

* Add some automatic tests for block-patterns

#### Testing instructions

```
cd apps/editing-toolkit/editing-toolkit-plugin/
yarn run test:php
```

Check the [README](https://github.com/Automattic/wp-calypso/blob/master/apps/editing-toolkit/README.md) for setup instructions

I'd written off the DOM tests as a non-starter, but there are some legit errors there, I just need to clear out the tag errors stepping from a lack of html5 support.

I suspect the validation errors are about my html-concatenation rather than real ones, but I need to investigate a bit more.

I've added some patterns that are obviously broken to highlight what these tests are _not_ catching.

See also: #43817
